src/h5web/vis-packs/core/shared/Html.tsx

```diff
@@ -70,20 +85,32 @@ const Html = forwardRef<HTMLDivElement, HtmlProps>((props, ref) => {
       if (parentElement) {
         el.remove();
       }
+
       ReactDOM.unmountComponentAtNode(el);
     };
-  }, [el, parentElement]); // eslint-disable-line react-hooks/exhaustive-deps
+  }, [el, parentElement]);
 
+  // Update size and overflow of container `div`
   useLayoutEffect(() => {
+    el.style.width = `${width}px`;
+    el.style.height = `${height}px`;
+    el.style.overflow = followCamera ? 'hidden' : 'visible';
+  }, [el, width, height, followCamera]);
```

Comment:
In this `useLayoutEffect`, I give and update the size of the outer `div` and I set `overflow: hidden` when `followCamera` is enabled. This is what allows annotations to be cropped on the edges of the canvas.

As mentioned, since the outer `div` now has a size, it has to have the `pointer-events: none` declaration (set in the `useState` hook above).